src/label.ts

```diff
@@ -9,16 +11,41 @@ export interface MatchExpression {
 
 export interface Selector {[l: string]: string | MatchExpression; }
 
-export function selectorToQueryString(selector: Selector): string {
-    const v: string[] = [];
-
-    _.forEach(selector, (value, label) => {
+export function selectorToString(selector: Selector, separator = ";"): string {
+    const parts = Object.entries(selector).map(([label, value]) => {
         if (typeof value === "string") {
-            v.push(label + "=" + value);
-        } else {
-            v.push(label + " "+value.operator+" (" + value.values.join(",") + ")");
+            return (label + "=" + value);
         }
+            return (label + " "+value.operator+" (" + value.values.join(",") + ")");
```

Comment:
Formatting seems off here; do we have a prettier config for this project?

Reply:
No. There is neither a configuration nor a script in the package.json or the dev dependency. I’d add this in another PR.